plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -174,15 +185,23 @@
     private static final String ICEBERG_MATERIALIZED_VIEW_COMMENT = "Presto Materialized View";
     public static final String DEPENDS_ON_TABLES = "dependsOnTables";
 
+    // Be compatible with views defined by the Hive connector, which can be useful under certain conditions.
+    private static final String TRINO_VIEW_COMMENT = HiveMetadata.PRESTO_VIEW_COMMENT;
+    private static final String TRINO_VERSION_NAME = HiveMetadata.PRESTO_VERSION_NAME;
+    private static final String TRINO_QUERY_ID_NAME = HiveMetadata.PRESTO_QUERY_ID_NAME;
+    private static final String TRINO_VIEW_EXPANDED_TEXT_MARKER = PRESTO_VIEW_EXPANDED_TEXT_MARKER;
```

Comment:
I would probably prefer to call this constants `PRESTO_*` as long as their contents are presto-specific, not trino-specific.

Reply:
Sure. Then does it make sense to redefine them here. Or just use constants from `HiveMetadata`?

Reply:
Redefining let's you pin a comment. It makes the intent more explicit.

i am fine with reuse too though